Rocket.Chat/Controllers/Chat/ChatControllerMessageCellProtocol.swift

```diff
@@ -72,6 +72,40 @@ extension ChatViewController: ChatMessageCellProtocol, UserActionSheetPresenter
         }
     }
 
+    func handleReadReceiptPress(_ message: Message, source: (UIView, CGRect)) {
+        guard let messageId = message.identifier else {
+            return
+        }
+
+        let controller = ReadReceiptListViewController()
+        controller.modalPresentationStyle = .popover
+        _ = controller.view
+
+        let api = API.current()
+        api?.fetch(ReadReceiptsRequest(messageId: messageId)) { response in
```

Comment:
Isn't it better to do this call inside of `ReadReceiptListViewController`?

Reply:
I agree here! 👍

Reply:
Done :+1: